pkg/server/helpers.go

```diff
@@ -52,6 +52,9 @@ const (
 	errorExitReason = "Error"
 	// oomExitReason is the exit reason when process in container is oom killed.
 	oomExitReason = "OOMKilled"
+
+	sandboxMetadataLabel ="io.cri-containerd.sandbox.metadata"
```

Comment:
Probably add another const section? Or put them into the second const section. :)